ansible/playbook/tasks/update.yml

```diff
@@ -124,6 +124,12 @@
     - name: Create working directory for Alertmanager
       file: path=/srv/alertmanager/data state=directory owner=pmm group=pmm
 
+    - name: Create working directory for VictoriaMetrics
+      file: path=/srv/victoriametrics/data state=directory owner=pmm group=pmm
+
+    - name: Create empty configuration file for VictoriaMetrics
+      file: path=/etc/victoriametrics-promscrape.yml state=touch owner=pmm group=pmm
```

Comment:
should this file have content `---`?

Reply:
It looks like it victoriametrics successfully reads empty yml file. @f41gh7 , could you confirm this?